.github/workflows/deploy-prod.yml

```
@@ -11,16 +11,48 @@ concurrency:
+      trigger_source: ${{ steps.set-outputs.outputs.trigger_source }}
+      release_version: ${{ steps.extract-release-info.outputs.version }}
+      release_ticket: ${{ steps.extract-release-info.outputs.ticket }}
+    steps:
+      - name: Set outputs
+        id: set-outputs
+        run: echo "trigger_source=${{ format('PR <{0}|#{1}> merged by <{2}|{3}>', github.event.pull_request.html_url, github.event.pull_request.number, format('{0}/{1}', github.server_url, github.actor), github.actor) }}" >> $GITHUB_OUTPUT
+
+      - name: Extract release info from branch
+        id: extract-release-info
+        if: startsWith(github.event.pull_request.head.ref, 'release/') || startsWith(github.event.pull_request.head.ref, 'hotfix-release/')
+        run: |
```

actionlint (1.7.7)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)